addons/l10n_ch/demo/res_partner_demo.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="res_partner_ch_qr" model="res.partner">
+        <field name="name">Easy Clean Lausanne</field>
+        <field eval="[(6, 0, [ref('base.res_partner_category_11')])]" name="category_id"/>
```

Comment:
If possible, we want to write these commands with `Command.set` instead (and it should be doable in xml/demo files)